### DISCUSSION TITLE
Request Demorado.

### DISCUSSION THREAD

**Discussion by pabloanizio** [HashLoad/horse#394] [Q&A]
Olá pessoal. Gostaria da ajuda de vcs aqui... Tenho um relatório de demora um pouco pra ser gerado.  Como vcs tratam esses requests mais demorados? alguém já passou por isso?

**Comment by ronaldobim** [comment]
Pablo eu penso que uma boa prática nestes casos é usar mensageria para
estes processos demorados, a ideia é que o serviço rest apenas encaminhe
uma mensagem para a mensageria(ActiveMQ / RabbitMQ), este por sua vez
possui um consumidor responsável por receber a mensagem e processar.

Em qua., 26 de jun. de 2024 às 13:40, Pablo Anízio ***@***.***>
escreveu:

> Olá pessoal. Gostaria da ajuda de vcs aqui... Tenho um relatório de demora
> um pouco pra ser gerado. Como vcs tratam esses requests mais demorados?
> alguém já passou por isso?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/HashLoad/horse/discussions/394>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALVZERUL5F6OCLH3OEN2KQLZJLVJDAVCNFSM6AAAAABJ6IMQK2VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHA3DMNJVHE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Ronaldo Cesar Bim
-------------------------------

Skype - ronaldobim
e-mail - ***@***.***

**Reply by pabloanizio** [reply]
Valeu Ronaldo. Obrigado pela resposta! Vou implementar dessa forma. Obrigado pela ajuda!
